src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_ListViewBase.cs

```diff
@@ -4112,6 +4112,61 @@ public async Task When_Footer_DataContext()
 			Assert.AreEqual(header2.DataContext.ToString(), header2.Text);
 		}
 
+
+		[TestMethod]
+		[RunsOnUIThread]
+		[DataRow("GridView")]
+		[DataRow("ListView")]
+		public async Task When_Header_With_HeaderTemplate_Only_One_Header_Created(string listViewBaseType)
+		{
+			var SUT = (ListViewBase)XamlReader.Load(
+				$"""
+				 <{listViewBaseType}
+				 	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+				 	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+				 	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+				 	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+				 	mc:Ignorable="d"
+				 >
+				 	<{listViewBaseType}.HeaderTemplate>
+				 		<DataTemplate>
+				 """
+				+
+				"			<TextBlock Text=\"{Binding}\" />"
+				+
+				$"""
+						</DataTemplate>
+					</{listViewBaseType}.HeaderTemplate>
+				</{listViewBaseType}>
+				""");
```

Comment:
Why are we concatenating strings like this? I think it should be possible to have this as a single interpolated raw string literal

Reply:
@ramezgerges @Xiaoy312 does https://github.com/unoplatform/uno/pull/14904 fix this particular PR ?

Reply:
14904 is a replacement of this pr